### PR TITLE
Map states from nl.nlziet app

### PR DESCRIPTION
**Pull request by berendhaan**
These were the only states I found that were usable to map to the playing / paused state. #12 

If you need any more information I'm happy to help!

### CODE REVIEW

**Comment by berendhaan** [comment]
@JeffLIrion I'm not sure why my change could trigger a failing linter. The original file already contained some very long lines.

File: androidtv/androidtv/base_androidtv.py [hunk 1]
```diff
@@ -171,6 +171,13 @@ def _update(
                 else:
                     state = constants.STATE_IDLE
 
+            # NLZIET
+            elif current_app == constants.APP_NLZIET:
+                if wake_lock_size == 1:
+                    state = constants.STATE_PAUSED
+                elif wake_lock_size == 2:
+                    state = constants.STATE_PLAYING
+
```
Comment (JeffLIrion) [inline androidtv/androidtv/base_androidtv.py:180]:
```suggestion

```

**Comment by JeffLIrion** [comment]
Please add unit tests to cover your changes. You can use this as a guide: https://github.com/JeffLIrion/python-androidtv/blob/9f0455d0c268926943c442ed4ae86b70a71b4ed0/tests/test_androidtv_sync.py#L888-L897

**Comment by berendhaan** [comment]
> 

Done :-) All checks are green.

**Pull request merged**